ui/components/Spacer.tsx

```diff
@@ -4,12 +4,20 @@ import { spacing } from "../typedefs/styled";
 type Props = {
   className?: string;
   margin?: keyof typeof spacing;
+  customMargin?: string;
   padding?: keyof typeof spacing;
+  customPadding?: string;
```

Comment:
Potential brilliant/terrible idea: could we do an array prop that roughly matches `padding` css formatting?:

```jsx
<Spacer padding={['small', 'small']} /> //Vertical-horizontal
```


```jsx
<Spacer padding={['small', 'small', 'big', 'big']} /> // control all 4 sides
```

That keeps the padding values constrained to known values, but provides more control.

Reply:
YES WE CAN and I want it because it's cool and fun. For now I just took the Spacer component out when I needed something specific but if this situation keeps happening that shouldn't be too hard to write